TeX_files/chapter01.tex

```diff
@@ -222,7 +222,7 @@ \section{Radar and remote sensing}
 \includegraphics[scale=0.2]{./graphics/SLAR-resolution}
 \caption{Diagram expressing the SLAR}
 \end{figure}
-\textbf{Synthetic aperture radar(SAR):} To improve the resolution in remote sensing, a technique called the synthetic aperture dish is used. For an antenna like a parabolic dish, the angular resolution is given as the wavelength divided by the size\footnote{For a parabolic dish antenna the diameter 'd' is used instead of length 'L'.See footnote 11} for the antenna $\Delta \theta = \frac{\lambda}{D}$. To get a fine resolution for the image in remote sensing, a very large aperture D is required. A large D cannot be easily created especially in moving vehicles and aircraft.\\
+\textbf{Synthetic aperture radar(SAR):} To improve the resolution in remote sensing, a technique called the synthetic aperture dish is used. For an antenna like a parabolic dish, the angular resolution is given as the wavelength divided by the size\footnote{For a parabolic dish antenna the diameter \textquotesingle d\textquotesingle \ is used instead of length \textquotesingle L\textquotesingle.See footnote 11} for the antenna, $\Delta \theta = \frac{\lambda}{D}$. To get a fine resolution for the image in remote sensing, a very large aperture D is required. A large D cannot be easily created especially in moving vehicles and aircraft.\\
```

Comment:
```suggestion
\textbf{Synthetic aperture radar(SAR):} To improve the resolution in remote sensing, a technique called the synthetic aperture dish is used. For an antenna like a parabolic dish, the angular resolution is given as the wavelength divided by the size\footnote{For a parabolic dish antenna the diameter \textquotesingle d\textquotesingle is used instead of length \textquotesingle L\textquotesingle. See footnote 11} for the antenna, $\Delta \theta = \frac{\lambda}{D}$. To get a fine resolution for the image in remote sensing, a very large aperture D is required. A large D cannot be easily created especially in moving vehicles and aircraft.\\
```